python/ray/autoscaler/azure/example-full.yaml

```diff
@@ -73,18 +73,13 @@ auth:
 # Note: the Azure Python SDK expects all parameter keys to be in slug_style
 #       the styles of parameter values are not changed
 head_node:
-    hardware_profile:
-        vm_size: Standard_D2s_v3
-    storage_profile:
-        os_disk:
-            create_option: FromImage
-            caching: ReadWrite
-        image_reference:
-            # List images https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage
-            publisher: microsoft-dsvm
-            offer: ubuntu-1804
-            sku: 1804-gen2
-            version: 20.02.01
+    azure_arm_parameters:
```

Comment:
suggest deleting head /worker node comments (lines 70-74 and 86-89 here). repeat for all relevant yamls